src/random/system.cr

```diff
@@ -0,0 +1,17 @@
+require "sys/random"
+
+# Generates random numbers from a secure source of the system.
+#
+# For example `arc4random` is used on OpenBSD, whereas on Linux it uses
+# `getrandom` (if the kernel supports it) and fallbacks on reading from
+# `/dev/urandom` on UNIX systems.
+class Random::System
+  include Random
```

Comment:
Both `Random::ISAAC` and `Random::MT1993` merely include `Random` and class methods seem to be specific to `Random`. What would `extend` bring?